src/Symfony/Component/Utf8/Exception/InvalidArgumentException.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Utf8\Exception;
+
+class InvalidArgumentException extends \InvalidArgumentException
```

Comment:
You should create an ExceptionInterface that acts as a marker for all exceptions coming from this component as done by other components.